test/e2e/framework/events/events.go

```diff
@@ -123,6 +129,57 @@ var _ = ginkgo.Describe("[sig-api-machinery] Events", func() {
 		}
 		framework.ExpectEqual(foundCreatedEvent, false, "should not have found test event after deletion")
 	})
+
+	ginkgo.It("should delete a collection of events", func() {
```

Comment:
Please implement e2e test outside of `test/e2e/framework`.
`test/e2e/framework` is a place to keep shared common functions for e2e tests.

Reply:
whoops, looks like there's a test already in here as well

Reply:
Both tests have been moved to `test/e2e/apimachinery/events.go`